selfie.c

```diff
@@ -566,6 +566,97 @@ initParser() {
 
 void emitLeftShiftBy(int b);
 void emitMainEntry();
+// *~*~ *~*~ *~*~ *~*~ *~*~ *~*~ *~*~ *~*~ *~*~ *~*~ *~*~ *~*~ *~*~
+// -----------------------------------------------------------------
+// ---------------------     O S   ---------------------------------
+// -----------------------------------------------------------------
+// *~*~ *~*~ *~*~ *~*~ *~*~ *~*~ *~*~ *~*~ *~*~ *~*~ *~*~ *~*~ *~*~
+
+
+// -----------------------------------------------------------------
+// ------------------------- Assignment 0 Linked List --------------
+// -----------------------------------------------------------------
+
+// Eg.: Size 2
```

Comment:
Think about the following data layout: Place the previous and next pointer at the beginning of the node. Then, the size of the node does not matter

+// +---------------+
+// | prev         |
+// +---------------+
+// | next         |
+// +---------------+
+// | payload starts here |
+// +---------------+
+// | and so on     |
+// +---------------+